src/v/config/bounded_property.h

```diff
@@ -215,9 +215,12 @@ class bounded_property : public property<T> {
       , _bounds(bounds)
       , _example(generate_example()) {}
 
+    using property<T>::set_value;
+
     void set_value(std::any v) override {
         property<T>::update_value(std::any_cast<T>(std::move(v)));
```

Comment:
Not really related, but I'm curious why this can't be just `property<T>::set_value(std::move(v))`

Reply:
sorry, not sure if i follow. What do you mean?